includes/api/class-wc-admin-rest-system-status-tools-controller.php

```diff
@@ -42,29 +42,4 @@ public function get_tools() {
 			)
 		);
 	}
-
-	/**
-	 * Actually executes a tool.
-	 *
-	 * @param  string $tool Tool.
-	 * @return array
-	 */
-	public function execute_tool( $tool ) {
```

Comment:
I don't think this is needed anymore since we run the tool through callbacks and only on the API.  Can anyone confirm this isn't used elsewhere?